package.json

```diff
@@ -5,7 +5,8 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "chain": "ts-node -O '{\"module\":\"commonjs\"}' chain.ts"
+    "chain": "ts-node -O '{\"module\":\"commonjs\"}' chain.ts",
+    "lint": "prettier './**/*.{ts,tsx,css,html,md}' --write"
```

Comment:
So you added this because you wanted to be able to run this over the entire repo to capture non-staged files?

Reply:
yup! I had added some additional functionality but my PR ended up being a lot more than the changes I was making due to the lint diffs :)